include/openthread/types.h

```diff
@@ -683,6 +683,8 @@ enum
     OT_CHANGED_THREAD_CHILD_REMOVED         = 1 << 11,  ///< Child was removed
     OT_CHANGED_IP6_MULTICAST_SUBSRCRIBED    = 1 << 12,  ///< Subscribed to a IPv6 multicast address
     OT_CHANGED_IP6_MULTICAST_UNSUBSRCRIBED  = 1 << 13,  ///< Unsubscribed from a IPv6 multicast address
+    OT_CPU_POWER_HI                         = 1 << 14,  ///< Hi CPU power is needed
```

Comment:
Of course we can adjust our solution. However we think that routines focused on computation power adjustment will be more generic than focused on Joiner/Commissioner state.